### PR TITLE
Revert "google-play-music-desktop-player: drop"

### PR DESCRIPTION
**Pull request by dotlambda**
###### Motivation for this change
The package is still useful for YouTube Music.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/google-play-music-desktop-player/default.nix [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+             "$out/share/google-play-music-desktop-player/Google Play Music Desktop Player"
+
+    wrapProgram $out/bin/google-play-music-desktop-player \
+      --prefix LD_LIBRARY_PATH : "$out/share/google-play-music-desktop-player" \
+      --prefix LD_LIBRARY_PATH : "${lib.makeLibraryPath deps}"
+  '';
+
+  meta = {
+    homepage = "https://www.googleplaymusicdesktopplayer.com/";
+    description = "A beautiful cross platform Desktop Player for Google Play Music";
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/google-play-music-desktop-player/default.nix:77]:
```suggestion
    description = "A beautiful cross platform Desktop Player for Google Play Music and YouTube Music";
```

**Comment by dotlambda** [comment]
@anna328p Would you like to be one of the maintainers for this package?

**Comment by anna328p** [comment]
> @anna328p Would you like to be one of the maintainers for this package?

Yes, I could do this. I already maintain an overlay for it.

**Review by anna328p — APPROVED** [review]
LGTM

**Pull request merged**